src/quo2/components/navigation/page_nav/view.cljs

```diff
@@ -56,7 +56,9 @@
    (cond
      ;; TODO: use account-switcher when available (issue #16456)
      (and support-account-switcher? (= content :account-switcher))
-     [rn/view {:style style/account-switcher-placeholder}]
+     [rn/touchable-opacity
+      {:style    style/account-switcher-placeholder
+       :on-press #(js/alert "Not implemented yet")}]
```

Comment:
this should just take an on-press and the not implemented should be set from the app code 👍

Reply:
Agree, but maybe is better to refactor once #16456 is done since it will need to be refactored anyway and use a dropdown variant with emoji